tests/playwright/sanity/modules/playing-cards/widget.test.js

```diff
@@ -0,0 +1,100 @@
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Playing Cards @playing-cards', () => {
+	let wpAdmin;
+	let editor;
+
+	test.beforeEach( async ( { page }, testInfo ) => {
+		wpAdmin = new WpAdminPage( page, testInfo );
+		editor = await wpAdmin.openNewPage();
+		await editor.addWidget( 'playing-cards' );
+	} );
+
+	test( 'Add widget', async () => {
+		const widget = await editor.getPreviewFrame().locator( EditorSelectors.playingCards.container );
+		await expect( widget ).toBeVisible( { timeout: 10000 } );
```

Comment:
I think that there's a default timeout of 8 seconds, might be wrong though

Reply:
fixed